Resources/Prototypes/_ES/Entities/Structures/Power/antimatter.yml

```diff
@@ -0,0 +1,193 @@
+      map: ["glow"]
+  - type: Appearance
+  - type: GenericVisualizer
+    visuals:
+      enum.ESAntimatterConverterVisuals.Draining:
+        glow:
+          True: { visible: true }
+          False: { visible: false }
+      enum.ESAntimatterConverterVisuals.Broken:
+        base:
+          True: { state: broken }
+          False: { state: icon }
+  - type: ESAntimatterConverter
+  - type: NodeContainer
+    examinable: true
+    nodes:
+      input:
+        !type:CableDeviceNode
+        nodeGroupID: HVPower
+  - type: PowerMonitoringDevice
+    group: Generator
+    loadNode: input
+    collectionName: negentropy
+    sprite: _ES/Structures/Power/negentropy.rsi
+    state: draining
+  - type: Battery
+    maxCharge: 5000000
+    startingCharge: 0
+  - type: BatteryDischarger
+  - type: PowerNetworkBattery
+    maxSupply: 350000
+    supplyRampTolerance: 350000
+  - type: PointLight
+    enabled: false
+    radius: 1.5
+    energy: 3
+    color: "#00AAFF"
+  - type: Transform
+    anchored: true
+  - type: Physics
+    bodyType: Static
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.30
+        density: 110
+        mask:
+        - MachineMask
+        layer:
+        - MachineLayer
+  - type: Anchorable
+  - type: Rotatable
+  - type: Pullable
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Repairable
+    fuelCost: 15
+    doAfterDelay: 3
+  - type: Damageable
+    damageContainer: StructuralInorganic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 50
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          collection: MetalBreak
+      - !type:DoActsBehavior
+        acts: ["Breakage"]
+
+- type: entity
+  parent: BaseStructure
+  id: ESAntimatterCapsule
+  name: antimatter capsule
+  description: A highly-experimental container used for transporting antimatter. Safe enough, short of throwing explosives at it. But you'd never do that...
+  components:
+  - type: Sprite
+    sprite: _ES/Structures/Power/antican.rsi
+    noRot: true
+    layers:
+    - state: icon
+    - state: unshaded
+      shader: unshaded
+  - type: InteractionOutline
+  - type: Clickable
+  - type: Transform
+    anchored: false
+    noRot: true
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: StrongMetallic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 100
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          collection: MetalBreak
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          ESAntimatter:
+            min: 1
+            max: 1
+      - !type:SpawnGasBehavior
+        gasMixture:
+          volume: 1000
+          moles:
+          - 0 # oxygen
+          - 0 # nitrogen
+          - 0 # carbon dioxide
+          - 100 # plasma
+          temperature: 373.15
```

Comment:
shouldnt the plasma be pumped in from outside to start it? didnt realize the can would have plasma in it too, is the intent that its like a bisected container